README.rst

```diff
@@ -111,6 +111,34 @@ Then add this line to your ``.zshrc``::
 
     source /usr/share/zsh/plugins/zsh-you-should-use/you-should-use.plugin.zsh
 
+`Home-Manager (Nix)`_
+
+Install the ``zsh-you-should-use`` (Nixpkgs_) package by adding it to ``programs.zsh.plugins``::
+
+    { pkgs, ... }:
+    {
+      programs.zsh.plugins = [
+        {
+          name = "you-should-use";
+          src = pkgs.zsh-you-should-use;
+          file = "share/zsh/plugins/you-should-use/you-should-use.plugin.zsh";
+        }
+      ];
+    }
+
+`NixOS`_
+
+Install the ``zsh-you-should-use`` (Nixpkgs_) package by adding it to ``programs.zsh.interactiveShellInit`` in your NixOS configuration::
+
+    { pkgs, ... }:
+    {
+      programs.zsh.interactiveShellInit = ''
+        source ${pkgs.zsh-you-should-use}/share/zsh/plugins/you-should-use/you-should-use.plugin.zsh
+      '';
+    }
```

Comment:
I dont use these so I dont know how to confirm this myself 🤔 Do you have a suggestion how I can verify these instructions? maybe I can run it within a docker container?

Reply:
For Home-Manager, you should be able to use [the Nix container](https://hub.docker.com/r/nixos/nix) and install a [standalone installation of HM](https://nix-community.github.io/home-manager/index.xhtml#sec-install-standalone). From there, include the provided snippet in your configuration (`.config/home-manager/home.nix`), then run `home-manager switch`.

The NixOS instructions are less easy to verify.